[FW][FW][FIX] portal_holidays: portal users didn´t see Leaves menu

fw-bot-adhoc opened the pull request:
Forward-Port-Of: ingadhoc/miscellaneous#117

Forward-Port-Of: ingadhoc/miscellaneous#118

roboadhoc commented:
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/miscellaneous/pull/119).

fw-bot-adhoc commented:
@fw-bot-adhoc @nicomacr cherrypicking of pull request ingadhoc/miscellaneous#118 failed.

stdout:
```
CONFLICT (modify/delete): portal_holidays/security/portal_holidays.xml deleted in HEAD and modified in 805dee8 ([FIX] portal_holidays: portal users didn´t see Leaves menu). Version 805dee8 ([FIX] portal_holidays: portal users didn´t see Leaves menu) of portal_holidays/security/portal_holidays.xml left in tree.
CONFLICT (modify/delete): portal_holidays/__manifest__.py deleted in HEAD and modified in 805dee8 ([FIX] portal_holidays: portal users didn´t see Leaves menu). Version 805dee8 ([FIX] portal_holidays: portal users didn´t see Leaves menu) of portal_holidays/__manifest__.py left in tree.

```

stderr:
```
18:09:52.833824 git.c:444               trace: built-in: git cherry-pick 805dee86bd0e579ed997b792ea7e3b6885824fe7
error: could not apply 805dee8... [FIX] portal_holidays: portal users didn´t see Leaves menu
hint: after resolving the conflicts, mark the corrected paths
hint: with 'git add <paths>' or 'git rm <paths>'
hint: and commit the result with 'git commit'
----------
status:

```

Either perform the forward-port manually (and push to this branch, proceeding as usual) or close this PR (maybe?).

In the former case, you may want to edit this PR message as well.

nicomacr commented:
@roboadhoc r+
